src/lib/utils/loadstor.h

```diff
@@ -382,219 +299,133 @@ inline constexpr void load_be(T out[], const uint8_t in[], size_t count) {
 }
 
 /**
-* Store a big-endian uint16_t
-* @param in the input uint16_t
-* @param out the byte array to write to
-*/
-inline constexpr void store_be(uint16_t in, uint8_t out[2]) {
-#if defined(BOTAN_TARGET_CPU_IS_BIG_ENDIAN)
-   typecast_copy(out, in);
-#elif defined(BOTAN_TARGET_CPU_IS_LITTLE_ENDIAN)
-   typecast_copy(out, reverse_bytes(in));
-#else
-   out[0] = get_byte<0>(in);
-   out[1] = get_byte<1>(in);
-#endif
-}
-
-/**
-* Store a little-endian uint16_t
-* @param in the input uint16_t
-* @param out the byte array to write to
+* Store a big-endian unsigned integer
```

Comment:
Same as with Load. Maybe: "Store an unsigned integer to bytes in big-endian order"?

Reply:
Here you have it right. I'm guessing that [your earlier comment](https://github.com/randombit/botan/pull/3707#discussion_r1398761064) was just a wording mixup.